### PR TITLE
various fixes and minor updates

### PR DESCRIPTION
**Pull request by BobTheBuidler**

### CODE REVIEW

**Review by banteg — APPROVED** [review]

File: yearn/prices/magic.py [hunk 1]
```diff
@@ -109,7 +109,8 @@ def find_price(token, block):
         return price * get_price(underlying, block=block)
 
     if not price:
-        logger.error("failed to get price for %s", token)
-        raise PriceError(f'could not fetch price for {token} at {block}')
+        symbol = contract(token).symbol()
```
[the remark this answers is not in the document]
Comment (BobTheBuidler) [reply yearn/prices/magic.py]: fix'd

File: yearn/prices/magic.py [hunk 2]
```diff
@@ -109,7 +109,8 @@ def find_price(token, block):
         return price * get_price(underlying, block=block)
 
     if not price:
```
[the remark this answers is not in the document]
Comment (BobTheBuidler) [reply yearn/prices/magic.py:111]: oof good catch, this one always catches me slipping

File: yearn/sentry.py [hunk 3]
```diff
@@ -21,5 +25,10 @@ def setup_sentry():
             before_send=before_send,
             debug=False,
             integrations=[ThreadingIntegration(propagate_hub=True)],
-            ignore_errors=[KeyboardInterrupt]
+            ignore_errors=[
+                KeyboardInterrupt, # these can be created when exiting a script with ctrl+c or when an exception is raised in a child thread. Ignore in both cases
+            ]
         )
+        set_tag('network',Network.label(chain.id))
```
[the remark this answers is not in the document]
Comment (BobTheBuidler) [reply yearn/sentry.py]: fix'd

File: yearn/prices/magic.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (BobTheBuidler) [reply yearn/prices/magic.py]: fix'd

**Pull request merged**